test/python/primitives/test_backend_estimator_v2.py

```diff
@@ -388,6 +388,78 @@ def test_diff_precision(self, backend, abelian_grouping):
         np.testing.assert_allclose(result[0].data.evs, [1.901141473854881], rtol=self._rtol)
         np.testing.assert_allclose(result[1].data.evs, [1.901141473854881], rtol=self._rtol)
 
+    @combine(backend=BACKENDS, abelian_grouping=[True, False])
```

Comment:
What is the value of looping over backends and abelian_grouping? Does it actually increase test coverage meaningfully?

Reply:
I removed the looping abelian_grouping and selected just one backend.